### ISSUE TITLE
Build broken with rusqlite 0.24

### ISSUE THREAD

**Issue by Progdrasil** [rust-db/refinery#134]
When depending on rusqlite 0.24 and refinery 0.3, theres a version conflict of the underlying libsqlite3-sys crate version. The dependency of `rusqlite = {version = ">= 0.23, < 0.25", optional = true}` in refinery-core does not seem to fix this.

Cargo.toml
```toml
# ...
[dependencies]
rusqlite = { version = "^0.24.1", features = ["uuid", "chrono", "serde_json", "load_extension"] }
refinery = { version = "^0.3.0", features = ["rusqlite"] }
# ...
```

Error
```
error: failed to select a version for `libsqlite3-sys`.
    ... required by package `rusqlite v0.23.0`
    ... which is depended on by `refinery-core v0.3.1`
    ... which is depended on by `refinery v0.3.0`
    ... which is depended on by mylib v0.2.2 (/home/rene/git/mylib)`
versions that meet the requirements `^0.18.0` are: 0.18.0

the package `libsqlite3-sys` links to the native library `sqlite3`, but it conflicts with a previous package which links to `sqlite3` as well:
package `libsqlite3-sys v0.20.1`
    ... which is depended on by `mylib v0.2.2 (/home/rene/git/mylib)`

failed to select a version for `libsqlite3-sys` which could resolve this conflict
```

**Comment by jxs** [comment]
Hi, yeah it has been updated on master, and I will release a new tiny today

**Comment by jxs** [comment]
there enough changes to a new minor release, so released `0.4` instead
